diff --git a/dist/components/index.js b/dist/components/index.js
index c7a55663ddb6b3f772d5dc8db87c7469d02df4d4..a8b8449bb00dff74e181fe0d4bb0eee7c8f168fd 100644
--- a/dist/components/index.js
+++ b/dist/components/index.js
@@ -901,7 +901,7 @@ var NavLink = /* @__PURE__ */ __name(({
   const { icons, classPrefix, enablePinning } = useNavConfig();
   const customIcons = useCustomIcons();
   const badge = useBadge(slug || "");
-  const isActive = !external && (pathname === href || pathname.startsWith(`${href}/`));
+  const isActive = !external && (pathname === href || (id !== "nav-dashboard" && pathname.startsWith(`${href}/`)));
   const extractedSlug = slug || id.replace("nav-global-", "").replace("nav-custom-", "").replace("nav-", "");
   let iconName = "file";
   if (customIcon) {
@@ -978,7 +978,7 @@ var PinnedItemLink = /* @__PURE__ */ __name(({ item, onUnpin, classPrefix, icons
   const badge = useBadge(item.slug);
   const customIcons = useCustomIcons();
   const isExternalLink = item.external || item.href.startsWith("http://") || item.href.startsWith("https://") || item.href.startsWith("//");
-  const isActive = !isExternalLink && (pathname === item.href || pathname.startsWith(`${item.href}/`));
+  const isActive = !isExternalLink && (pathname === item.href || (item.slug !== "dashboard" && pathname.startsWith(`${item.href}/`)));
   let iconName = "file";
   if (item.icon) {
     iconName = icons[item.icon] || item.icon;
